projects/v2/src/app/modules/root/root.component.ts

```diff
@@ -216,18 +220,18 @@ export class RootComponent implements OnInit, OnDestroy {
       }
       else if (selectedOrgans && playground !== 'true') {
         store.dispatch(new UpdateMode('vis'));
-        this.sheet =  SHEET_CONFIG.find(i => i.name === 'some');
+        this.sheet =  this.SHEET_CONFIG.find(i => i.name === 'some');
```

Comment:
Also try to clean up some of the formatting issues. For instance, this has two spaces after the equals instead of one.